apps/ledger-live-mobile/.env.android.prerelease

```diff
@@ -1,7 +1,6 @@
 APP_NAME="Ledger Live"
 SENTRY_DSN=https://ea730d6c531f40679e7306c888f7fbc5@o118392.ingest.sentry.io/6723478
 ANALYTICS_TOKEN=jfUZbw28ig8JpEi9DZpTUc21dKUKu1e3
-GOOGLE_SERVICE_INFO_NAME="GoogleService-Info-Production"
```

Comment:
This is unused and it's confusing everyone. Unfortunately we cannot _easily_ configure Google services (thus Firebase) dynamically with a simple environment variable.